modules/ROOT/pages/spike-control-reference.adoc

```diff
@@ -92,9 +82,45 @@ When you apply the policy to your API from the UI, the following parameters are
 | Method & Resource conditions | The option to add configurations to only a select few or all methods and resources of the API | Yes
 |===
 
-The Spike Control policy does not perform quota enforcement. The configuration parameters are restricted to protect the API and backend. Therefore, to ensure maximum performance, configure these parameters to the lowest possible value. 
+To enforce request quotas, see the xref:rate-limiting-sla-based-policy.adoc[Rate-Limiting SLA policy] and xref:rate-limiting.adoc[Rate Limiting policy].
+
+== FAQ
+
+*When does the sliding window start?*
+
+The window starts with the first request after the policy is successfully applied.
+
+*What type of window does the algorithm use?*
+
+The algorithm uses a sliding window by monitoring the transactions for the amount of time for which you configure in the policy.
+
+*What happens when the quota is exhausted?*
+
+When the request quota is exhausted, the Spike Control policy queues the request and retries as configured. The request is rejected if no quota is remaining after several retries. Quota for other requests is only available after the first request has elapsed the time in millisecconds as configured in the policy. Requests are processed in cycles using the configured time limit in this way.
+
+*Can I configure multiple windows?*
+
+No. The Spike Control policy ensures that the backend server does not serve more requests than it can handle. If you need accountability, use either Rate-Limiting SLA or Rate Limit policies instead.
+
+[[response-header]]
+*What does each response header mean?*
+
+Each response header has information about the current state of the request:
+
+* X-Ratelimit-Remaining: The amount of available quota
+* X-Ratelimit-Limit: The maximum available requests per window
+* X-Ratelimit-Reset: The remaining time, in milliseconds, until the oldest request ages enough to fall outside of the sliding window. If quota is still available, then this header is set to 0 (zero), as the algorithm can still serve new requests.
```

Comment:
"as" > "because," if that is what you really mean

Reply:
Also, bullet-list format:

```suggestion
* X-Ratelimit-Reset: The remaining time, in milliseconds, until the oldest request ages enough to fall outside of the sliding window. 
+
If quota is still available, then this header is set to 0 (zero), as the algorithm can still serve new requests.
```

Reply:
Done! Thanks!